bin/start-uc-cli-in-docker

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+set -ex;
+
+container_version="0.1.0"
+container_name="unitycatalog-cli"
+network_name="unitycatalog_network"
+network_details=$(docker network ls --format '{{json .}}' | jq -r --arg NETWORK_NAME "$network_name" 'select(.Name == $NETWORK_NAME) | .Name')
+container_details=$(docker container ls --format '{{json .}}' | jq -r --arg CONTAINER_NAME "$container_name" 'select(.Names == $CONTAINER_NAME) | .Names')
+container_runing=$(docker ps --format '{{json .}}' | jq -r --arg CONTAINER_NAME "$container_name" 'select(.Names == $CONTAINER_NAME) | .Names')
```

Comment:
typo - i think you mean `container_running`?

Reply:
Done :+1: